src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exceptions.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final Properties DB_PROPERTIES;
+    private static final String URL = "jdbc:mysql://localhost:3306/mate";
+    private static final String USER = "root";
+    private static final String PASSWORD = "password";
```

Comment:
Storing the database username and password as plain text in the source code is not secure. Consider using environment variables or a configuration file to store sensitive information securely.